adafruit_bitbangio.py

```diff
@@ -76,7 +84,7 @@ def deinit(self):
 class I2C(_BitBangIO):
     """Software-based implementation of the I2C protocol over GPIO pins."""
 
-    def __init__(self, scl, sda, *, frequency=400000, timeout=1):
```

Comment:
It isn't implemented yet, but I'm pretty sure `timeout` will be allowed to be `float`, so let's type annotate it as such.